### ISSUE TITLE
More support for golden card tracking

### ISSUE THREAD

**Issue by WrHyiU** [MGK82/PackTracker#4]
The pity timers on golden cards still suffer from very small sample sizes. It's also very interesting to track this for your own packs. It would be nice to include a more clear interface for golden cards and the max / average distance for golden cards of each rarity.

**Comment by MGK82** [comment]
It's on the road map, but I don't see it as a priority feature atm.

First idea was to expand the statistic, the window is small anyway. What do you think?

**Comment by WrHyiU** [comment]
Yeah expanding the statistics window would be fine. For this suggestion I was thinking there could be an optional / second statistics window just for golden cards. 

One side question though, where is the data stored locally? I might want to ask some friends for their data so I can look at it.

**Comment by MGK82** [comment]
I don't think a 6th window would help the plugin, especially when most windows are pretty small.

To your other question:
The data is stored in your HDT appdata. You can open your plugin folder with HDT, then one folder up and PackTracker\ or just %appdata%\HearthstoneDeckTracker\PackTracker\

It's a plain xml (History.xml)

Be careful if you edit it:
1. You falsify your data. You only harm yourself
2. When you make a mistake, PackTracker will deny to load it and start without a history. When you open a pack then, it will store this new history and overwrite your old one.
